module.json

```diff
@@ -9,6 +9,8 @@
   "license": "Apache-2.0",
   "extraIncludes": [],
   "dependencies": {
+    "mbed-hal-frdm-k64f":"sg-/mbed-hal-frdm-k64f#crc-uuid",
```

Comment:
Why do we need this here ? Was it because of IP address problem ? This is now fixed and merged so we should not need this.